src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -195,3 +198,62 @@
               :on-press            (fn []
                                      (rf/dispatch [:hide-bottom-sheet])
                                      (when on-press (on-press)))}]))]])))
+
+(defn get-reaction-authors-accessibility-id
+  [reaction-type]
+  (-> reaction-type
+      constants/reactions
+      name
+      (str "-reaction-count")))
+
+(defn reaction-authors
+  [reaction-authors show-reaction-author-list?]
+  (let [selected-tab (reagent/atom (first (keys reaction-authors)))]
+    [:f>
+     (fn []
+       (rn/use-effect (fn []
+                        (fn []
+                          (reset! show-reaction-author-list? false)))
+                      [reaction-authors])
+       [:<>
+        [quo/tabs
+         {:style          style/tabs
+          :size           32
+          :on-change      #(reset! selected-tab %)
+          :default-active @selected-tab
+          :data           (mapv (fn [[reaction-type author-details]]
+                                  ^{:key (str reaction-type)}
```

Comment:
Is the `reaction-type` really unique among other `reaction-authors`? If the keys clash the rendering performance will take a hit. Wouldn't it be better to combine the `reaction-type` with something else?

Reply:
You're right, It is so not unique 😆.
Gonna make it unique

Reply:
I thought the same at first, But this renders the tabs that render at the top of the view.
Which I think they're unique because they are constants

Reply:
Right, so each tab is a different reaction type, so it should be fine. Good point